src/2.lighting/1.colors/1.light_cube.fs

```diff
@@ -3,5 +3,5 @@ out vec4 FragColor;
 
 void main()
 {
-    FragColor = vec4(1.0); // set alle 4 vector values to 1.0
+    FragColor = vec4(1.0); // set all 4 vector values to 1.0
 }
```

Comment:
I personally prefer files to have a newline at eof, but I try not to fix that while making PRs like this one